src/codegen/utilities/attribute_helpers.py

```diff
@@ -240,14 +243,17 @@
 }
 
 
-def get_attributes(metadata, class_name):
+def get_attributes(metadata, class_name, additional_class_name=""):
     """Converts the scrapigen metadata into a list of attributes."""
     attributes_metadata = []
     for group_name, attributes in metadata["attributes"].items():
         for id, attribute_data in attributes.items():
             if (
                 "python_class_name" in attribute_data
-                and attribute_data["python_class_name"] == class_name
+                and (
+                    attribute_data["python_class_name"] == class_name
+                    or attribute_data["python_class_name"] == additional_class_name
+                )
```

Comment:
I think you should update scrapigen to generate the right `python_class_name` and revert this `additional_class_name` change.